cpp/src/groupby/new_groupby.cu

```diff
@@ -190,3 +192,230 @@ gdf_error gdf_group_by(gdf_column* in_key_columns[],
 
   return gdf_error_code;
 }
+
+
+
+// thrust::device_vector set to use rmmAlloc and rmmFree.
+template <typename T>
+using Vector = thrust::device_vector<T, rmm_allocator<T>>;
+
+
+namespace {
```

Comment:
If this is in an anonymous namespace, does that mean you don't want it to be part of the external libcudf API?

I think it would be useful to expose this function in our API.